backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell
+
+import com.typesafe.config.Config
+import cromwell.core.{WorkflowId, WorkflowOptions}
+import wdl4s._
+
+package object backend {
+
+  /**
+    * For uniquely identifying a job which has been or will be sent to the backend.
+    */
+  case class BackendJobDescriptorKey (callFqn: String, index: Option[Int], attempt: Int)
+
+  /**
+    * For passing to a BackendWorkflowActor at job execution, recovery or aborting time
+    */
+  case class BackendJobDescriptor(key: BackendJobDescriptorKey, call: Call, locallyQualifiedInputs: CallInputs)
```

Comment:
I think this is wrong - the Engine cannot evaluate `locallyQualifiedInputs` - only the backend can